### DISCUSSION TITLE
Removing SARCO as the payment token

### DISCUSSION THREAD

**Discussion by adamgall** [sarcophagus-org/sarcophagus-v2-specs#10] [Ideas]
# What's the problem

Embalmers are required to pay for Sarcophagus services (to archaeologists) using the SARCO token. This has a couple of drawbacks:

* Embalmers need to go acquire SARCO from AMMs or centralized exchanges, for no other reason than to use the software.
* SARCO isn't a well-known token (yet), so embalmers need to consciously perform exchange rates to understand if they are comfortable with the costs involved.
* Archaeologists set their payment fees in SARCO (three different fees: `fees_per_byte` for Arweave storage costs, `min_bounty` for minimum bounty limits, and `min_digging_fee` for minimums paid on each re-wrap). So as the price of SARCO is volatile, it requires Archaeologists to continuously update their SARCO rates to stay competitive, which adds friction to archaeologists due to the gas costs involved with updating payment rates.

# Let's fix it

As an idea to float and discuss, what if we replaced "payments in SARCO" with "ETH" or "stable coins"? This would allow both embalmers and archaeologists to negotiate in currencies they're more familiar with, and are easier to "value".

Some things to continue thinking about and digging into:

* Should archaeologists still place their bonds in SARCO? How does that affect the "contract" between embalmers and archaeologists, if they misbehave and the protocol slashes one of them. Presumably risk-skew would occur if the relative value of SARCO (archaeologist bond) and ETH/stables (embalmer "bond") changes significantly.

**Comment by dacebt** [comment]
What if the model allowed for the choice from (supported list of tokens which would including SARCO) and incentivize SARCO payments? This would allow for Archaeologist to have the ability to have payments in any supported currency while also allowing for a path and use of SARCO as payment? Grain of salt as this is not my area of expertise, but from a user and arch standpoint makes for an easy way to get started.

**Reply by adamgall** [reply]
This is certainly an option.

I think it would increase complexity a lot though.

Archaeologists would have to specify which currencies they support, and their rates for each (unless we used an on-chain oracle to normalize everything into USD, which is a possibility).

How would archaeologist bonds work with multiple payment currency options? Same issue applies even with just a single non-SARCO payment option, but gets trickier to reason with when there are many currency options.

All of this is assuming that these multiple currencies are on the protocol level, not just the UI level. Making these changes on the UI level would make it easier for embalmers for sure, but wouldn't have any effect on archaeologist UX.

**Comment by sethhrbek** [comment]
Some other thoughts (somewhat random, not making any specific argument) -- 

For Arch bonding, currently in v1:
1. If price of SARCO goes up, the bond value attached to any given sarcophagus (locked bond) goes up
2. Inversely, SARCO price down =  locked bond value on sarcophagus goes down

If SARCO goes down, the overall value of the arch's not-yet-unwrapped sarcophagi goes down, thus the incentive to complete the unwrapping goes down.

The Archaeologist could adjust their digging (or other) fees to try to capture some of the skew during a rewrap event. But there is the chance the embalmer would not want to pay the increased fees.
